front-end/src/types.ts

```diff
@@ -140,10 +139,3 @@ export interface ReactionMapFields {
 	userNames: string[]
 }
 
-export interface AccountsDetails {
```

Comment:
adding types in an external file only makes sense when this type is reused elsewhere. It's not the case here.